docs/3.0.0-beta.x/guides/auth-request.md

```diff
@@ -0,0 +1,172 @@
+
+**User 2**
+
+- **username**: reader
+- **email**: reader@strapi.io
+- **password**: strapi
+- **role**: Reader
+
+## Login as a Reader
+
+To login as a user your will have to follow the [login documentation](../plugins/users-permissions.md#login).
+
+Here is the API route for the authentication `/auth/local`.
+
+You have to request it in **POST**.
+
+:::: tabs
+
+::: tab axios
+
+```js
+import axios from 'axios';
+
+const { data } = await axios.post('http://localhost:1337/auth/local', {
+  identifier: 'reader@strapi.io',
+  password: 'strapi',
+});
+
+console.log(data);
+```
+
+:::
+
+::: tab Postman
+
+If you are using **Postman** for example you will have to set the `body` as `raw` with the `JSON (application/json)` type.
+
+```json
+{
+  "identifier": "reader@strapi.io",
+  "password": "strapi"
+}
+```
+
+:::
+
+::::
+
+The API response contains the **user's JWT** in the `jwt` key.
+
+```json
+{
+    "jwt": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNTc2OTM4MTUwLCJleHAiOjE1Nzk1MzAxNTB9.UgsjjXkAZ-anD257BF7y1hbjuY3ogNceKfTAQtzDEsU",
+    "user": {
+        "id": 1,
+        "username": "reader",
+        ...
+    }
+}
+```
+
+You will have to store this `JWT` in your application, it's important because you will have to use it the next requests.
+
+### Fetch articles
+
+Let's fetch Articles you created.
+
+To do so, you will have to fetch `/articles` route in **GET**.
+
+```js
+import axios from 'axios';
+
+const { data } = await axios.get('http://localhost:1337/articles');
+
+console.log(data);
+```
+
+Here you should receive a **403 error** because you are not allowed, as Public user, to access to the **articles**.
+
+You should use the `JWT` in the request to say that you can access to this data as **Reader user**.
+
+```js
+import axios from 'axios';
+
+const { data } = await axios.get('http://localhost:1337/articles', {
+  headers: {
+    Authorization:
+      'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNTc2OTM4MTUwLCJleHAiOjE1Nzk1MzAxNTB9.UgsjjXkAZ-anD257BF7y1hbjuY3ogNceKfTAQtzDEsU',
+  },
+});
+
+console.log(data);
+```
+
+And tada you have access to the data.
+
+### Create an Article
+
+To do so, you will have to request the `/articles` route in **POST**.
+
+```js
+import axios from 'axios';
+
+const {data} = await axios
+  .get('http://localhost:1337/articles',  {
+    data: {
+      title: 'my article'
+      content: 'my super article content'
+    },
+    headers: {
+      'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNTc2OTM4MTUwLCJleHAiOjE1Nzk1MzAxNTB9.UgsjjXkAZ-anD257BF7y1hbjuY3ogNceKfTAQtzDEsU'
+    }
+  });
+
+console.log(data);
+```
+
+If you request this as a **Reader user**, you will receive a **403 error**. It's because the **Reader role** not have access to the `create` function of the **Article** Content Type.
+
+To fix that you will have to login with the **Author user** and use his `JWT` into the request to create an **Article**.
```

Comment:
```To fix that you will have to login with the **Author user** and use its `JWT` into the request to create an **Article**.```

(trying to keep it gender neutral here, idc personally but with all the spark in the tech world best to avoid his, her, him, ect)

Reply:
You are right! My English mistake.

Reply:
No problem, the rest looked quite good.